src/components/Asset/AssetActions/Download/index.tsx

```diff
@@ -388,7 +388,10 @@ export default function Download({
             <div className={styles.confettiContainer}>
               <SuccessConfetti
                 success={`You successfully bought this ${
-                  asset.metadata.type
+                  asset?.metadata?.additionalInformation?.saas?.paymentMode ===
```

Comment:
suggestion(blocking): it would be cleaner to bind this message to the service being a `saas` in general. We should check if the asset is marked as `saas` and if it is display `"service"` over `metadata.type`